ui/pages/confirmations/hooks/useSignatureSecurityAlertResponse.ts

```diff
@@ -0,0 +1,19 @@
+import { useSelector } from 'react-redux';
+import { SecurityAlertResponse } from '../types/confirm';
+
+type SignatureSecurityAlertResponsesState = {
+  metamask: {
+    signatureSecurityAlertResponses: Record<string, SecurityAlertResponse>;
+  };
+};
+
+function useSignatureSecurityAlertResponse(securityAlertId?: string) {
```

Comment:
since this is `useSignatureSecurityAlertResponse` instead of `useCurrentSignatureSecurityAlertResponse`, we are passing the securityAlertId